src/lib.rs

```diff
@@ -64,10 +64,10 @@ fn update() -> Result<(), Box<::std::error::Error>> {
 }
 ```
 
-Separate utilities are also exposed:
+Separate utilities are also exposed (**NOTE**: the following example _requires_ the `archive-tar` feature):
 
 ```rust
-
+#[cfg(feature = "archive-tar")]
```

Comment:
is there a way to hide these attributes from showing up in the rendered example? It's usually the `# ` prefix, but not sure if that works with cfg-attrs?

Reply:
Yes it even works there! :D Fixed in bfb799a0fc331e02b567561c00b83581.